examples/modules/heat/pages/opus/README.adoc

```diff
@@ -289,51 +290,12 @@ ICs.
 As the periodicity helps to dissipate the heat in both directions of the PCB, the reference results
 are attended to be lower than the results obtained with Feel++ on these testcases.
 
-++++
-<div id="plotly_test1a"></div>
-<script type="text/javascript">
-  Plotly.d3.csv("https://girder.math.unistra.fr/api/v1/file/5afea86fb0e957402704804a/download",
-    function(err,rows) {
-      var data = [{
-        name: 'IC2 reference results of test 1a',
-        type: 'scatter',
-        x: feelpp_unpack(rows,'T'),
-        y: feelpp_unpack(rows,'Ref-test1a-s1'),
-        showlegend: true,
-        line: { color: '#FF99BB' }
-      },{
-        name: 'IC2 example results of test 1a',
-        type: 'scatter',
-        x: feelpp_unpack(rows,'T'),
-        y: feelpp_unpack(rows,'Results-test1a-s1'),
-        showlegend: true,
-        line: { color: '#CC3333' }
-      },{
-        name: 'Output reference results of test 1a',
-        type: 'scatter',
-        x: feelpp_unpack(rows,'T'),
-        y: feelpp_unpack(rows,'Ref-test1a-s2'),
-        showlegend: true,
-        line: { color: '#BB99FF' }
-      },{
-        name: 'Output example results of test 1a',
-        type: 'scatter',
-        x: feelpp_unpack(rows,'T'),
-        y: feelpp_unpack(rows,'Results-test1a-s2'),
-        showlegend: true,
-        line: { color: '#3333CC' }
-      }];
-
-      var layout = {
-        title: 'Temperature'
-      };
-      Plotly.plot(plotly_test1a,data,layout,{ showLink: false });
-
-      feelpp_add_redimensionable_plot(plotly_test1a);
-    }
-  );
-</script>
-++++
+:test-name: test 1a
+:test-key: test1a
+[plotly,{url-plotly-data},subs=+attributes]
+....
+include::example$opus/plotly-test-definition.js[]
+....
```

Comment:
Include the common definition and define the value of `test-name` and `test-key`